api/src/twitterBot/bree.ts

```diff
@@ -0,0 +1,22 @@
+import Bree from "bree";
+
+//This is helper necessary for bree to work w typescript
+function typescript_worker() {
```

Comment:
this will work only in dev mode, in prod the code is compiled to js so you probably ned to run the worked with ts-node only in dev and make sure that for prod everything is working (run yarn build + yarn start for prod mode)